app/concepts/fec/exporter/xml.rb

```diff
@@ -60,7 +60,7 @@ def build(journals)
             end
           end
         end
-        builder.to_xml
+        builder.to_xml.gsub("&#8217;", "'")
```

Comment:
https://www.rubydoc.info/github/sparklemotion/nokogiri/Nokogiri/XML/Node#to_xml-instance_method
You should set the encoding in the to_xml method :-) U+8217 is the Unicode code for the `'` character.

Reply:
`builder.to_xml(encoding: 'ISO-8859-15')`
doesn't work either, unfortunately